include/model/network-simple-multicast.hpp

```diff
@@ -196,6 +196,10 @@ class SimpleMulticastNetwork : public Network
 
   // Ugly abstraction-breaking probes that should be removed.
   std::uint64_t WordBits() const;
+  std::uint64_t FillLatency() const override { assert(is_specced_); return specs_.fill_latency.Get(); };
```

Comment:
My apologies for sending this back again, but we are trying to avoid inline function definitions as far as possible (although I know there are some instances in the codebase), both because of code-hygiene reasons as well as for cleaner interfacing with PyTimelop. Could you please move the definitions into the .cpp file? Thanks.

Reply:
Yes, will make sure to separate the declaration and the definition for future commits.